docs/page-objects/inputs.rst

```diff
@@ -62,6 +62,10 @@ define as inputs for a page object class, including:
 -   :class:`~web_poet.page_inputs.browser.BrowserHtml`, an HTML rendering of
     the `Document Object Model`_ of a web page.
 
+-   :class:`~web_poet.page_inputs.browser.BrowserResponse`, which includes URL,
+    status code and :class:`~web_poet.page_inputs.browser.BrowserHtml`
+    of a rendered web page.
```

Comment:
What about replacing rather than adding it to the list?

```suggestion
-   :class:`~web_poet.page_inputs.browser.BrowserResponse`, which includes URL,
    status code and :class:`~web_poet.page_inputs.browser.BrowserHtml`
    of a rendered web page.
```

Reply:
Makes sense!